frontend/public/components/cluster-settings/_cluster-operator.scss

```diff
@@ -0,0 +1,29 @@
+.co-cluster-operator__message {
+  @include co-break-word;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  height: 42px;
+  margin-bottom: 5px;
+}
+
+.co-cluster-operator__message:after {
+  background: linear-gradient(to right, rgba(255, 255, 255, 0), rgba(255, 255, 255, 1) 50%);
```

Comment:
I believe we should have other existing styles we should be able to reuse for the gradient

@sg00dwin @rhamilto ?

Reply:
Per @rhamilto, these styles were removed from other tables in #1031. We shouldn't be using them here. (I know it's in the design, but I'm assuming it's there because that's what other tables previously used.)

Reply:
I'll reset back to the original then.